wallets/provider-solflare-snap/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@rango-dev/provider-solflare-snap",
+  "version": "0.30.0",
+  "license": "MIT",
+  "type": "module",
+  "source": "./src/index.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "node ../../scripts/build/command.mjs --path wallets/provider-metamask",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "clean": "rimraf dist",
+    "format": "prettier --write '{.,src}/**/*.{ts,tsx}'",
+    "lint": "eslint \"**/*.{ts,tsx}\" --ignore-path ../../.eslintignore"
+  },
+  "dependencies": {
+    "@rango-dev/signer-solana": "^0.26.0",
+    "@rango-dev/wallets-shared": "^0.30.0",
+    "@solana/web3.js": "^1.67.2",
```

Comment:
I think you could remove dependency to solana/web3 and use our signer for handling transactions

Reply:
Done